templates/logstash.service.j2

```diff
@@ -0,0 +1,28 @@
+# {{ ansible_managed }}
```

Comment:
Add `# {{ ansible_managed  }}` string to any template managed by Ansible. Read more at [config docs](https://jpmens.net/2020/09/29/using-ansible-managed/).

Note: don't add any dynamic value into that variable